### PR TITLE
Changes to German localisation

### PR DESCRIPTION
**Pull request by DonSqueak**
Pull request for whatever will come next; also, three changes got lost somewhere, so I recommit them here.

### CODE REVIEW

File: CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff [hunk 1]
```diff
@@ -660,9 +659,8 @@
         <note>Do not translate</note>
       </trans-unit>
       <trans-unit id="Known For" xml:space="preserve">
-        <source>Known For</source>
-        <target state="translated">Bekannt aus</target>
-        <note/>
+        <source>Known For</source><target state="needs-review-translation">Bekannt für</target>
```
Comment (DonSqueak) [inline CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff:662]: Also an old change

File: CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff [hunk 2]
```diff
@@ -1020,8 +1018,7 @@
         <note/>
       </trans-unit>
       <trans-unit id="Rating" xml:space="preserve">
-        <source>Rating</source>
-        <target state="translated">Bewertung</target>
+        <source>Rating</source><target state="new">Freigabe</target>
```
Comment (DonSqueak) [inline CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff:1021]: Also old

**Comment by DonSqueak** [comment]
Git is killing me btw. Half of it I don't understand, the other half doesn't seem to work :D

**Comment by cliss** [comment]
Welcome to git. ;)

https://github.com/cliss/callsheet-localizations/pull/17#issuecomment-2108914667

**Pull request merged**